rfqmsg/sell_accept.go

```diff
@@ -94,13 +93,7 @@ func newSellAcceptFromWireMsg(wireMsg WireMessage,
 // ShortChannelId returns the short channel ID associated with the asset sale
 // event.
 func (q *SellAccept) ShortChannelId() SerialisedScid {
-	// Given valid RFQ message id, we then define a RFQ short chain id
-	// (SCID) by taking the last 8 bytes of the RFQ message id and
-	// interpreting them as a 64-bit integer.
-	scidBytes := q.ID[24:]
-
-	scidInteger := binary.BigEndian.Uint64(scidBytes)
-	return SerialisedScid(scidInteger)
+	return q.ID.Scid()
```

Comment:
note for commit message:

This code should be relevant for generating RFQ scids within range: [lnd/aliasmgr/aliasmgr.go](https://github.com/lightningnetwork/lnd/blob/master/aliasmgr/aliasmgr.go#L508)

Reply:
Yeah, I have that code in a WIP branch ([`lnd-0-19-staging-rebased`](https://github.com/lightninglabs/taproot-assets/compare/lnd-0-19-staging-rebased) already: https://github.com/lightninglabs/taproot-assets/blob/97be885dcfa6d35b19a150580cd1dd0d921403c7/rfqmsg/messages.go#L41